in-progress/8754-slashing-and staking.md

```diff
@@ -0,0 +1,357 @@
+
+Therefore the Executing Validators who can't download all `TxObject` and proof data for any given block should initiate a data withholding slashing proposal, which works in exactly the same way as the invalid state root slashing proposal.
+
+
+### 2. ETH Congestion
+
+Provers may not be able to post proofs if the L1 is congested or is experiencing an inactivity leak. Instead of automatic slashing of the prover bond, the validator set can vote to "unslash" the provers. 
+
+The prover's bond sits in escrow (or a separate contract) for $t=4$ weeks. During this period, the prover can plead their case and convince the validators to vote to unslash them. A validator must initiate a vote to unslash the prover and submit to L1 for verification on time, otherwise the bond gets slashed. 
+
+Since "Ethereum was congested" is a subjective statement, two ways we could implement this:
+
+1) Make $t$ larger, giving the prover enough time to convince the validators to manually update their software to vote to unslash the prover. This could be done on the forums or validator community channels for example. 
+2) Add an environment variable `max_eth_base_fee` that clients agree to implement. If `base_gas_fee > max_gas_fee` at the proof submission deadline time, then validator nodes automatically sign votes to unslash prover.
+
+They are largely equivalent except that the second requires less coordination.
+
+### PoG Non-slashable offences
+
+We would like to slash the following actions but this would require vast changes to the p2p and/or would still be susceptible to attacks. Still including them here for brevity but these offenses are NOT slashable in the implementation.  
+
+### 1. Sequencers don't accept EpochProofQuotes
+
+If no EpochProofQuote is accepted within $C=13$ slots and the epoch reorgs, validators check for whether sequencers could have accepted an actionable EpochProofQuote during their turn. 
+
+An actionable quote is one where:
+i) The timestamp of receiving it was before the $C=13$th slot of the epoch. 
+ii) The epoch information is correct. 
+iii) The prover had the required bond in escrow. 
+iv) The `basisPointFee` was within the allowable range. 
+
+Validators check for whether they saw such an actionable `EpochProofQuote`. If they did, they initiate a proposal to slash all committee members that could've accepted the quote but did not. 
+
+Why is this hard to implement?
+- We're designing S&P coordination to be out-of-protocol. The above assumes provers send quotes via the p2p.
+- Provers currently can send quotes even without having the necessary bond. 
+- Provers could send the quotes to a partition of the network, excluding the proposers who can activate the quote. This leads to timing games.
```

Comment:
It is unclear to me what is meant by "timing games" here.

Reply:
I think it refers to this type of scenario: You make a case that you submitted the proposal on time, and there is timestamp / log evidence that you did so, but you submitted it late enough that it did not propagate to the sequencers who would have sent it??

I dont think the protocol should be forgiving in these scenarios, Provers should send quotes via p2p or spray it to many validators to avoid this happening to them.